build.sh

```diff
@@ -72,11 +85,37 @@ install_rpms() {
     rpm -q grubby && dnf remove -y grubby
     # Further cleanup
     dnf clean all
-
 }
 
-make_and_makeinstall() {
 
+install_rpms_rhel() {
```

Comment:
I should have seen this earlier, but I think we can just collapse both versions of `install_rpms()` into a single function that just uses `yum`.  `/usr/bin/yum` is just a symlink to `dnf-3` on F29.

Reply:
That sounds like a cleaner approach than duplicating the code.

Might require some additional organization/renaming of existing `deps.txt`, `build-deps.txt`, and `Dockerfile` to indicate they are for Fedora builds